modules/openapi-generator/src/main/resources/C-libcurl/model-body.mustache

```diff
@@ -526,8 +526,8 @@ fail:
     char* decoded = base64decode({{{baseName}}}->valuestring, strlen({{{baseName}}}->valuestring));
     decoded_str_{{{baseName}}}->data = malloc(strlen(decoded) - 1);
     if (!decoded_str_{{{baseName}}}->data) {
-		goto end;
-	}
+        goto end;
+    }
```

Comment:
@michelealbano thanks for fixing the formatting in the templates.

Another way is to use a code formatter such as https://clang.llvm.org/docs/ClangFormat.html to format the output.

Reply:
I didn't know it works for templates, I thought it got confused by the syntax.
Nice to know.